.golangci.yml

```diff
@@ -330,6 +362,9 @@ linters:
       - linters:
           - staticcheck
         text: 'SA1019: "sigs.k8s.io/cluster-api/(.*)" is deprecated: This package is deprecated and is going to be removed when support for v1beta1 will be dropped.'
+      - linters:
+          - staticcheck
+        text: "s.scope.ControlPlaneLoadBalancer is deprecated"
```

Comment:
We should remove this ignore and address the actual method change to the new one in a later pr.

Reply:
Can you create an issue, if there's not one? :pray:

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/5795